File: HelpSoftware/Help/Help.Web/Web/src/app/EF/Tag.ts
export class Tag {
  ID_Tag: number;
  TagName: string;
  Created: Date;
  TagDescription: string;
  ColorString: string;

  deserialize(input) {
    this.ID_Tag = input.ID_Tag;
    this.TagName = input.TagName;
    this.Created = input.Created;
    this.TagDescription = input.TagDescription;
    this.ColorString = input.ColorString;
    return this;
  }
}

File: HelpSoftware/Help/Help.EF/Entitys/Tag.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Help.EF
{
  public class Tag : IHelpEntity
  {
    [Key]
    public virtual int ID_Tag { get; set; }

    public virtual string TagName { get; set; }

    public virtual DateTime Created { get; set; }

    public virtual string TagDescription { get; set; }

    //WerteBereich
    public virtual string ColorString { get; set; }

    [Timestamp]
    public virtual byte[ ] TimeStamp { get; set; }

    public string GetEntityName()
    {
      return "Tag";
    }

    public int GetID()
    {
      return ID_Tag;
    }

    public bool GetWriteHistoryEntry()
    {
      return true;
    }
  }
}